src/simpla-admin.html

```diff
@@ -112,8 +135,16 @@
       }
 
       /**
+       * Attach singleton to window on created
+       * @return {[type]} [description]
+       */
+      ready() {
```

Comment:
Probably better to put this into `attached`. Polymer's `ready` is a bit weird, rely's on components in the `local` DOM to be attached, `attached` is more in line with standard platform.